btk/utils.py

```diff
@@ -0,0 +1,124 @@
+                    the files.
+        measure_name (str): Name of the measure function for which you
+                    want to load the files
+        n_batch (int): Number of blends in the batch you want to load
+
+    Returns:
+        Dictionnary containing the detection catalogs, the segmentations
+        and the deblended images.
+    """
+    measure_results = {}
+    for key in ["segmentation", "deblended_images"]:
+        try:
+            measure_results[key] = np.load(f"{path}_{measure_name}_{key}.npy", allow_pickle=True)
+        except FileNotFoundError:
+            print(f"No {key} found.")
+    catalog = []
+    for j in range(n_batch):
+        catalog.append(
+            Table.read(
+                f"{path}_{measure_name}_detection_catalog_{j}",
+                format="ascii",
+            )
+        )
+    measure_results["catalog"] = catalog
+    return measure_results
+
+
+def load_metrics_results(path, measure_name):
+    """Load results exported from a MetricsGenerator.
+
+    Args:
+        path (str): Path to the files. Should be the same as the save_path
+                    which was provided to the MetricsGenerator to save
+                    the files.
+        measure_name (str): Name of the measure function for which you
+                    want to load the files
+
+    Returns:
+        Dictionnary containing the detection catalogs, the segmentations
+        and the deblended images.
+    """
+    metrics_results = {}
+    for key in ["detection", "segmentation", "reconstruction"]:
+        try:
+            metrics_results[key] = np.load(
+                f"{path}_{measure_name}_{key}_metric.npy", allow_pickle=True
+            )
+        except FileNotFoundError:
+            print(f"No {key} metrics found.")
+
+    metrics_results["galaxy_summary"] = Table.read(
+        f"{path}_{measure_name}_galaxy_summary",
+        format="ascii",
+    )
+    return metrics_results
+
+
+def load_all_results(path, surveys, measure_names, n_batch):
+    """Load results exported from a MetricsGenerator.
+
+    Args:
+        path (str): Path to the files. Should be the same as the save_path
+                    which was provided to the MetricsGenerator to save
+                    the files.
+        surveys (list): Names of the surveys for which you want to load
+                        the files
+        measure_names (list): Names of the measure functions for which you
+                    want to load the files
+        n_batch (int): Number of blends in the batch you want to load
+
+    Returns:
+        The three dictionnaries corresponding to the results.
+    """
+    blend_results = {}
+    for key in ["blend_images", "isolated_images", "blend_list"]:
+        blend_results[key] = {}
+    measure_results = {}
+    metrics_results = {}
+    for s in surveys:
+        blend_results_temp = load_blend_results(path, s)
+        for key in ["blend_images", "isolated_images", "blend_list"]:
```

Comment:
redundant with line 111

Reply:
Ok, but I'm curious to what difference it makes.

Reply:
Just a common refactoring practice.  
For one it specifies this is constant (and therefore a tuple would be even better than the list I proposed).  
And also, if in the future you want to add or remove an element, you have to do it once and limit the possibilities for typos.